README.md

```diff
@@ -1,2 +1,70 @@
-# portfolioskeleton1
-Mobile Portfolio Skeleton task 1
+
+# Project Name: Portfolio Skeleton Project
+
+> header and and headline section 
+Created new branch according to GitHub flow rules
+Added a .gitignore file
+setup a linters for html and css and made sure that files not relevant are ignored by git.
+Added a descriptive README file 
+Added a header and headline section in the index.html file  of the project 
+Added css styling to the html sections 
+
+## Built With
+- HTML and CSS
+- Visual Studio Code -Electron framework
+- Technologies used
+
+## Getting Started
+
+**This project include linters for HTML,CSS, and Open Source Document
+.**
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Node package Manager(NPM)
+Follow set up instructions from Node Official Website https://nodejs.org/en/download/
+Text Editor: VS-Code or any other Code editor 
+
+### Setup
+Simple Setup  folder
+  portfolioskeleton1
+  package.json
+  index.html
+  styles.css
+
+### Install
+
+### Usage
+npx hint .
+npx stylelint "**/*.{css,scss}"
+
+### Run tests
+
+### Deployment
+## Authors
+
+👤 **Richard W.K Chileya**
+
+- GitHub: [@richardchileya](https://github.com/richardchileya)
+- Twitter: [@Richard Chileya](https://twitter.com/richardchileya)
+- LinkedIn: [Richard Chileya](https://linkedin.com/in/richardchileya)
+
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Your README looks great 👏. In addition to what you have done, I think it **would be better if you remove this ending note** because it does not make your README look professional 😃 👍